lib/uniq.js

```diff
@@ -4,7 +4,11 @@
  * @return: An array containing unique elements
  */
 function uniq(array) {
-  // Your code goes here!
+  if (Array.isArray(array)) {
+    return [...new Set(array)];
+  }
+
+  throw "array is now an Array";
```

Comment:
just a typo in your "throw"- it should read "array is not an Array"

Reply:
oops. I'll fix it in a minute.